### DISCUSSION TITLE
Can't open GeoTIFF files or vectors

### DISCUSSION THREAD

**Discussion by P-Courtney** [stangandaho/Extractor#6] [Q&A]
This plugin seems brilliant, but it won't let me open GeoTIFFs. 
<img width="648" alt="image" src="https://github.com/stangandaho/Extractor/assets/63449444/7c0e2343-b702-4ce8-aaf3-6a4bdcdc969c">

**Comment by stangandaho** [comment]
Thanks for sharing your experience.
Please, what QGIS version are you using?

On Thu, 14 Mar 2024 at 15:24, P-Courtney ***@***.***> wrote:

> This plugin seems brilliant, but it won't let me open GeoTIFFs.
> image.png (view on web)
> <https://github.com/stangandaho/Extractor/assets/63449444/7c0e2343-b702-4ce8-aaf3-6a4bdcdc969c>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/stangandaho/Extractor/discussions/6>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN5EECMEVF6C5J3T6VVWTJLYYGQJ7AVCNFSM6AAAAABEWDL62WVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGM3TEOJYG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

*Stanislas Mahussi GANDAHO*,

*Data Scientist*

*Protected Areas Manager*

*Naben <https://re-agro.org/>'s Member*

**Comment by P-Courtney** [comment]
V3.36.0. 

Thanks for the reply!

Lastly, I can copy and paste the directory into the box, but then I can't select the statistic, sum, mean etc.

**Comment by stangandaho** [comment]
Okay help me to get your log messages. To do so;
- Go to *View* menu -> *Panels* -> *Log messages*
- Check Log messages
- Close Extractor and open again.
- Then try to do your operation.
If an information appears in *Messages*, *Python warnings* or *Python
errors* tab in your Log messages panel, let me know.

On Fri, 15 Mar 2024 at 21:35, P-Courtney ***@***.***> wrote:

> V3.36.0.
>
> Thanks for the reply!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/stangandaho/Extractor/discussions/6#discussioncomment-8806774>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN5EECMOKJEOUU6WTP2Z7MLYYNERRAVCNFSM6AAAAABEWDL62WVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQMBWG43TI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

*Stanislas Mahussi GANDAHO*,

*Data Scientist*

*Protected Areas Manager*

*Naben <https://re-agro.org/>'s Member*
